tests/harness/python/BUILD

```diff
@@ -0,0 +1,16 @@
+load("@my_deps//:requirements.bzl", "requirement")
+load("@io_bazel_rules_python//python:python.bzl",  "py_binary")
+
+
+py_binary(
+    name = "python-harness",
+    srcs = ["harness.py"],
+    deps = [
+        "//tests/harness:python-harness-proto",
+        "//tests/harness/cases:python",
+           requirement("validate-email"),
```

Comment:
do we still need a requirements.txt file if these are specified here?

Reply:
yup I think so, I believe bazel first installs the packages using the `requirements.txt` and then we need to specify them here. This was my source: https://github.com/bazelbuild/rules_python